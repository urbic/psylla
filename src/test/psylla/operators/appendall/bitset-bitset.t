bitset dup ( 3 2 7 5 ) appendall dup
bitset dup ( 7 5 11 13 ) appendall
appendall ?
