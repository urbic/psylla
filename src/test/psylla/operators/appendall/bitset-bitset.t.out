%bitset=00110101000101%
